docker-compose.integration-test.yml

```diff
@@ -1,13 +1,103 @@
 version: "3"
 
 services:
+
```

Comment:
What is the idea behind this new yml file? It seems to copy the production yaml pretty closely.

Reply:
The idea of this PR is to create a separate full local experimenter stack specifically just for the integration tests rather than use the dev environment one.  The problem we ran into was on someones local environment, their dev environment might have some arbitrary settings (including the BUGZILLA_HOST) that don't match what the tests expect, so even on a 'clean' master checkout, the tests were failing.  So I changed it to make its own separate isolated experimenter environment which mirrors the way prod is setup, except without a couple things it doesn't need specifically for development (like shared volumes) and in its own isolated namespace `experimenter_integration` so that it doesn't collide with the dev environment.  This way the state of your dev environment shouldn't affect whether the integration tests pass.